home_3_2.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"unicode/utf8"
+)
+
+func main() {
+
+	sliceWords := []string{"one", "thr22", "two", "three", "thr33"}
+	fmt.Println(max(sliceWords))
+
+	sliceNum := []int64{0, 1, 2, 4, 5, 67, 8}
+	fmt.Println(reverse(sliceNum))
+
+}
+
+func max(slice []string) string {
+	lenSrt := 0
+	var elem string
+	maxLen := 0
+	for _, v := range slice {
+
+		lenSrt = utf8.RuneCountInString(v)
+
+		if maxLen < lenSrt {
+			maxLen = lenSrt
+			elem = v
```

Comment:
Do we really need to store length and string in a separate variables ? I suggest to store only `elem` string and use `len(elem)` to get the length of string.